Dockerfile

```diff
@@ -1,3 +1,35 @@
+# This stage extracts all the pom.xml files.
+# It'll get rebuilt with any source change, but that's OK.
+FROM openjdk:11-jdk AS poms
```

Comment:
openjdk is deprecated (https://hub.docker.com/_/openjdk)

Is `eclipse-temurin:11-jdk-noble` the closest similar?